src/app/content/practiceQuestions/reducer.spec.ts

```diff
@@ -1,32 +1,7 @@
-import { locationChange } from '../../navigation/actions';
 import * as actions from './actions';
 import reducer, { initialState } from './reducer';
 
 describe('practice questions reducer', () => {
-  it('keeps modal open on location change if modal query is present', () => {
```

Comment:
you could add check for `locationChange` to make sure that we are handling it properly

Reply:
Is it really needed? 
`locationChange` reducer is very simple right now:
```
case getType(locationChange):
   return {
      ...state,
      currentQuestionIndex: null,
      questionAnswers: {},
      questions: [],
      selectedSection: null,
   };
```

Reply:
I feel like this might be something that will easly break if we update somehow this reducer in the future and we will not get idea why something might not work
This test will be easy to write and just might become usefull in the future
this is not blocking comment